osu.Game/Localisation/UserInterfaceStrings.cs

```diff
@@ -110,9 +110,9 @@ public static class UserInterfaceStrings
         public static LocalisableString ModSelectHotkeyStyle => new TranslatableString(getKey(@"mod_select_hotkey_style"), @"Mod select hotkey style");
 
         /// <summary>
```

Comment:
And at this point, you can combine the two usages of background blur into a single translation. I'd probably just use the gameplay one for now, rather than adding this one:
 https://github.com/ppy/osu/blob/bed25ff60e92b87d212a15281d65556e555e587c/osu.Game/Localisation/GameplaySettingsStrings.cs#L52-L55

Reply:
okay, fixed in [acb42f7](https://github.com/ppy/osu/pull/22382/commits/acb42f7d1265fe8ab7a72d302a4af6d7e12d7bb9)